src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/fragment/FeedFragment.java

```diff
@@ -31,29 +31,22 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        this.mFeedItems = new ArrayList<FeedItem>();
```

Comment:
No. We only add elements to the feed items list now in ``onPostExecute`` of the AsyncTask (this is invoked on the UI thread).